contracts/GnosisSafe.sol

```diff
@@ -50,7 +50,7 @@ contract GnosisSafe
     );
 
     uint256 public nonce;
-    bytes32 public domainSeparator;
+    bytes32 private _deprecatedDomainSeparator;
```

Comment:
Why don't remove the old `domainSeparator`? Do we need to keep that variable in the proxy storage?

Reply:
yes else the other variables are messed up. The storage locations are calculated based on the order or variables

So in this case it would mess with `signedMessages` and `approvedHashes` ... not the worst, but I would avoid it

Reply:
https://docs.soliditylang.org/en/v0.8.1/internals/layout_in_storage.html